tests/testsuite/lto.rs

```diff
@@ -192,33 +231,47 @@ fn complicated() {
         )
         .build();
     p.cargo("build -v --release")
-        // normal deps and their transitive dependencies do not need object
-        // code, so they should have linker-plugin-lto specified
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name dep_normal2 [..]-C linker-plugin-lto[..]`",
-        )
-        .with_stderr_contains("[..]`rustc[..]--crate-name dep_normal [..]-C linker-plugin-lto[..]`")
-        // build dependencies and their transitive deps don't need any bitcode,
-        // so embedding should be turned off
-        .with_stderr_contains("[..]`rustc[..]--crate-name dep_build2 [..]-C embed-bitcode=no[..]`")
-        .with_stderr_contains("[..]`rustc[..]--crate-name dep_build [..]-C embed-bitcode=no[..]`")
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name build_script_build [..]-C embed-bitcode=no[..]`",
-        )
-        // proc macro deps are the same as build deps here
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name dep_proc_macro2 [..]-C embed-bitcode=no[..]`",
-        )
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name dep_proc_macro [..]-C embed-bitcode=no[..]`",
-        )
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name foo_bin [..]--crate-type bin[..]-C lto[..]`",
-        )
-        .with_stderr_contains(
-            "[..]`rustc[..]--crate-name test [..]--crate-type cdylib[..]-C lto[..]`",
+        // - normal deps and their transitive dependencies do not need object
+        //   code, so they should have linker-plugin-lto specified
```

Comment:
Let's revert this one.  I don't trust `does_not_contain` much but pairing them with easily read `contains` that make it easy to see when one needs updating, helps.